README.md

```diff
@@ -64,3 +64,11 @@ Run `bundle install` to let the bundler automatically install all required packa
 Go grab a cup of coffee while this is running, it could take a while. Next, when you're in the `events` folder, run `bundle exec jekyll serve`. This will create a local server for you, which can be accessed by typing `http://localhost:4000/events/`
 
 Ta-da!
+
+## Instructions before adding images on the website
```

Comment:
These instructions do not make sense for the general documentation, since nobody outside the org is allowed to make edits that involve images. Also, websites like tinypng and smallpdf are honeypots from a security point of view, so we shouldn't be recommending them.

Reply:
The instructions for are internal, but this seemed to be the best place to document these. Anyone who's updating the event websites with photos needs these instructions, otherwise, the images don't render correctly.
Right, so which services would you recommend to include here?

cc: @StephanieBr